tests/app/notifications/test_rest.py

```diff
@@ -14,6 +16,51 @@
 from tests.app.db import create_api_key, create_notification
 
 
+@pytest.mark.usefixtures(
+    "client",
+    "sample_template",
+)
```

Comment:
I'm assuming this is here due to the note below about the use of the function-scoped fixtures, and that it's forcing the method to load them beforehand ([reading over the docs on it](https://docs.pytest.org/en/stable/reference/reference.html#pytest-mark-usefixtures))?

Not pointing out anything wrong, I just wanted to double check I'm understanding correctly! 🙂